### ISSUE TITLE
Migration documentation incomplete

### ISSUE THREAD

**Issue by kingprimex** [codership/documentation#207]
Where it is specified `$ mysqldump -u root -p --skip-create-options --all-databases > migration.sql` should also be a note that this export can break everything in the new database. For example, if you have any fields as PK with auto_increment, that would not be imported correctly. There could be a lot of extra infos from the db that would not be copied to the galera cluser. IMO the preferred way to export from MyISAM to INNODB would be a normal export and manually change the ENGINE. Cheers.

**Comment by kirjaamo** [comment]
Can I fix this by adding the note below to Migrating to Galera Cluster -> Data Migration, where the command above is mentioned:

Note!
If the migration does not correctly import all data, you can try to export the data and manually change the database engine.

**Comment by byte** [comment]
@kirjaamo where do we specify this? esp. the "--skip-create-options" ? do we even have a document re: migrating from myisam to innodb? can i read the whole thing in context?

**Comment by kirjaamo** [comment]
Here:
https://galeracluster.com/documentation/html_docs_20210916-0928-possibility-to-install-galera-on-freebsd-with-packages/training/tutorials/migration.html?highlight=data%20migration

**Comment by kirjaamo** [comment]
But we do not have a document re: migrating from myisam to innodb

**Comment by kirjaamo** [comment]
@byte You are right, we do not have a document re: migrating from myisam to innodb. Can I use the link below to create that?

https://support.severalnines.com/hc/en-us/articles/212426983-Migrate-MyISAM-tables-to-INNODB-using-mysqldump
